sdk/python/kfp/utils.py

```diff
@@ -0,0 +1,61 @@
+def use_secret(secret_name=None, volume_name=None, secret_volume_mount_path=None, env_variable=None, secret_file_path_in_volume=None):
```

Comment:
Would you mind adding some test for this function? I just realized we don't have unittest for `use_gcp_secret` either but it's definitely not the best practice...

Reply:
Sure I can add tests :)

Reply:
I have added the a test :)